ti/instrumentation/utils.go

```diff
@@ -264,6 +267,151 @@ func getChangedFiles(ctx context.Context, workspace string, log *logrus.Logger,
 	return res, nil
 }
 
+// addBazelFilesToChangedFiles takes a list of files and removes bazel build files and adds java files listed in target src globs
+func addBazelFilesToChangedFiles(ctx context.Context, workspace string, log *logrus.Logger, res []ti.File, bazelFileCount string) ([]ti.File, []string, error) {
+	var moduleList []string
+	threshold, err := strconv.Atoi(bazelFileCount)
+	if err != nil {
+		return res, moduleList, fmt.Errorf("bazelFileCount not set correctly in ticonfig.yml, expecting number found character %v ", bazelFileCount)
```

Comment:
This is a client call validation not the method's (since you chose to make it a util method).